From 2800496b0f170e5e7564fdd8031857611f8a1732 Mon Sep 17 00:00:00 2001
From: "Kevin F. Haggerty" <haggertk@lineageos.org>
Date: Mon, 23 Apr 2018 22:37:03 -0600
Subject: [PATCH 2/3] Revert "klte-common: Enable legacy mediaserver"

This reverts commit aee12a34df50bd5484a8d67d87a1a2abe0543acd.

* No Samsung msm8974 device is going to develop an OSS camera HAL
  at this point. Move this common stuff to msm8974-common.

Change-Id: Ib0bb18fe819e2ebbb39c9b278ccf687444f65488
---
 BoardConfigCommon.mk | 1 -
 system.prop          | 2 --
 2 files changed, 3 deletions(-)

diff --git a/BoardConfigCommon.mk b/BoardConfigCommon.mk
index d54aae6..90e65fb 100644
--- a/BoardConfigCommon.mk
+++ b/BoardConfigCommon.mk
@@ -34,7 +34,6 @@ BOARD_HAVE_SAMSUNG_BLUETOOTH := true
 TARGET_BOOTLOADER_BOARD_NAME := MSM8974
 
 # Camera
-TARGET_HAS_LEGACY_CAMERA_HAL1 := true
 USE_DEVICE_SPECIFIC_CAMERA := true
 
 # Extended Filesystem Support
diff --git a/system.prop b/system.prop
index de5476f..9722f8d 100644
--- a/system.prop
+++ b/system.prop
@@ -34,8 +34,6 @@ ro.qc.sdk.izat.premium_enabled=0
 ro.qc.sdk.izat.service_mask=0x0
 
 # Media
-media.stagefright.legacyencoder=true
-media.stagefright.less-secure=true
 persist.media.treble_omx=false
 
 # NFC
-- 
2.17.0

